lib/themes/dosomething/paraneue_dosomething/scss/content/_finder.scss

```diff
@@ -213,7 +213,7 @@
   }
 
   .no-result {
-    background: $off-black asset-url("images/placeholder.png") center center;
+    background: $off-black;
```

Comment:
Can we make this `background-color`? Would make it easy to customize adding to it if needed without having to override a bunch of the default properties the shorthand otherwise adds...